Orange/widgets/visualize/owsieve.py

```diff
@@ -43,8 +43,9 @@ def __init__(self, data, attr1, attr2):
         self.probs_x = self.observed.sum(axis=0) / self.n
         self.probs_y = self.observed.sum(axis=1) / self.n
         self.expected = np.outer(self.probs_y, self.probs_x) * self.n
-        self.residuals = \
-            (self.observed - self.expected) / np.sqrt(self.expected)
+        with np.errstate(divide="ignore", invalid="ignore"):
+            self.residuals = \
+                (self.observed - self.expected) / np.sqrt(self.expected)
```

Comment:
Can we not divide by zero instead of ignoring warnings?

Reply:
No, it's a vector. I could create a mask, but it would be annoying. It's simpler this way.